src/server_manager/messages/en.json

```diff
@@ -4,9 +4,9 @@
   "aws-lightsail-firewall-0": "Navigate to the {openLink}Amazon Lightsail{closeLink} instances screen.",
   "aws-lightsail-firewall-1": "Click the instance on which you want to host Outline.",
   "aws-lightsail-firewall-2": "Navigate to the 'Networking' tab.",
-  "aws-lightsail-firewall-3": "In the 'Firewall' section, click 'Add another'.",
-  "aws-lightsail-firewall-4": "Set 'Application' value to 'All TCP+UDP'.",
-  "aws-lightsail-firewall-5": "Click 'Save'.",
+  "aws-lightsail-firewall-3": "Scroll down to 'IPv6 firewall', then click 'Add rule'.",
```

Comment:
What about IPv4?

Reply:
AWS'll auto-create the same rules for IPv4 by default.

Reply:
Other ways also exist for setting up the rule, but I believe these steps are quick and easy.